Cargo.toml

```diff
@@ -4,17 +4,26 @@ version = "0.1.10"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/prisma/connection-string"
 documentation = "https://docs.rs/connection-string"
-description = "Connection string parsing in Rust"
+description = "Connection string parsing in Rust (and WebAssembly)"
 readme = "README.md"
 edition = "2018"
 keywords = []
 categories = []
 authors = [
-    "Yoshua Wuyts <yoshuawuyts@gmail.com>"
+    "Yoshua Wuyts <yoshuawuyts@gmail.com>",
+    "Julius de Bruijn <julius@nauk.io>",
 ]
 
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = ["-Oz", "--enable-mutable-globals"]
+
 [features]
+wasm = ["wasm-bindgen"]
 
 [dependencies]
+wasm-bindgen = { version = "0.2", optional = true }
 
 [dev-dependencies]
+
+[lib]
+crate-type = ["cdylib", "lib"]
```

Comment:
Also not completely fresh on this, but will this affect native builds?

Reply:
No. The `lib` option here is the one our native builds use, `cdylib` is needed for wasm. This is from the Rust wasm playbook.